components/landing/landingButtons.tsx

```diff
@@ -3,23 +3,27 @@ import getDownloadLink from '../../lib/platform';
 const githubUrl = 'https://github.com/unitystation/unitystation';
 const discordInviteUrl = 'https://discord.com/invite/tFcTpBp';
 const githubReleases = 'https://github.com/unitystation/stationhub/releases/latest';
+const patreonUrl = 'https://www.patreon.com/unitystation';
+import { faCloudDownload } from '@fortawesome/free-solid-svg-icons';
 
 import {useEffect, useState} from 'react';
 
 
-
 const LandingButtons = () => {
     const [downloadLink, setDownloadLink] = useState(githubReleases);
 
     useEffect(() => {
         getDownloadLink().then((link) => setDownloadLink(link));
     }, []);
 
-    return (<div className={'flex flex-wrap justify-center mt-8 gap-4'}>
-            <Button filled={true} text={'Download'} linkTo={downloadLink}/>
-            <Button filled={false} text={'Github'} linkTo={githubUrl}/>
-            <Button filled={false} text={'Discord'} linkTo={discordInviteUrl}/>
-        </div>)
+    return (
+        <><div className={'flex flex-wrap justify-center mt-8 gap-4'}>
```

Comment:
I don't like having two elements at the same level. Use the IDE auto formatting.

Reply:
wdym?

Reply:
I should note that I have no idea what I'm doing with this sort of stuff because I'm not a web dev, I just try things and see what sticks.

Reply:
```suggestion
        <>
            <div className={'flex flex-wrap justify-center mt-8 gap-4'}>
            ...
```